### ISSUE TITLE
Add hidden attribute when it's hidden

### ISSUE THREAD

**Issue by Haroenv** [Haroenv/holmes#72]
Should be the default since it then won't be spoken by screen readers (it also won't with display none or opacity hidden)

Fixed in #67 but still needs a good name
